andela/valinor-ah-backend
# #161291020 Implement time to read an article

#### What does this PR do?
Add the time to read an article to the article table

#### Description of Task to be completed?
- add a readTime column to the article table
- calculate the time taken to read an article and add to the article table

#### How should this be manually tested?
- clone the repo, checkout to this branch `feature/161291020/calculate-time-to-read`
- set up your local db and .env variables
- sign up a user using the sign up route 
- create an article using the create article route
- see the calculated reading time in the response or query the db 

#### Any background context you want to provide?
The `readTime` value is in microseconds. Transformations can be done on the front-end.

#### What are the relevant pivotal tracker stories?
[161291020](https://www.pivotaltracker.com/story/show/161291020)

#### Screenshots (if appropriate)
Articles table columns:
<img width="227" alt="screen shot 2018-11-14 at 3 28 45 pm" src="https://user-images.githubusercontent.com/31810086/48489028-5f49bc00-e822-11e8-9b92-4d23788fbd56.png">

Read time of a created article:
<img width="985" alt="screen shot 2018-11-14 at 3 29 37 pm" src="https://user-images.githubusercontent.com/31810086/48489035-61137f80-e822-11e8-9553-d6998d602565.png">

#### Questions:
N/A
